minecraft_servers/brickcraft/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+    "server_name": "brickcraft",
+    "nice_name": "BrickCraft",
+    "direct_ip": "brickcraft.eu",
+    "supported_languages": [
+        "de"
+    ],
+    "social": {
+        "discord": "https://discord.gg/ThCt8Pq",
+        "teamspeak": "craftergang.de",
+        "tiktok": "craftergang",
+        "youtube": "https://www.youtube.com/@craftergang",
+        "instagram": "craftergang"

```

Comment:
Are you sure that these are your social profiles?

Reply:
fixed